ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -69,11 +69,6 @@ is_chown_forbidden() {
   return $?
 }
 
-is_vboxsf_mountpoint() {
-  grep -q "/app vboxsf" /proc/mounts
-  return $?
-}
-
```

Comment:
use case would be im using native linux (no vm at all) so the only change i need when in development is the uid/gid fix.

root@fd0b312508fa:/# findmnt /app

TARGET SOURCE                                 FSTYPE OPTIONS
/app   /dev/sda1[/home/david/projects/forces] ext4   rw,relatime,errors=remount-ro,data=ordered

by being specific to vbox/osxfs etc those env variables are never set. how about starting with a check like in the PR then running the edge cases to account for chown based on specific filesystem types.

am happy to adjust the PR if it would be accepted.